ui/apps/notifications/src/components/newComponents/NotificationsPage.tsx

```diff
@@ -0,0 +1,305 @@
+        },
+        property: 'POST_MENTION',
+      },
+      createdAt: Date.now(),
+    },
+    {
+      id: '3',
+      body: {
+        value: {
+          author: {
+            name: 'Dr. Cat',
+            userName: 'thedrCat',
+            ethAddress: '0x003410490050000320006570034567114572000',
+            avatar: { url: 'https://placebeard.it/360x360' },
+          },
+          follower: {
+            name: 'Dr. Flynn',
+            userName: 'thedrflynn',
+            ethAddress: '0x003410490050000320006570034567114572000',
+            avatar: { url: 'https://placebeard.it/360x360' },
+          },
+          postID: '01f3st44m5g3tc6419b92zyd21',
+        },
+        property: 'POST_MENTION',
+      },
+      createdAt: Date.now(),
+      read: true,
+    },
+  ]; // notifReq.data;
+
+  const unreadNotifications = allNotifications?.filter(notif => notif.read === undefined);
+
+  const markAsRead = useMarkAsRead();
+
+  const handleAvatarClick = (profilePubKey: string) => {
+    navigateTo?.({
+      appName: '@akashaorg/app-profile',
+      getNavigationUrl: navRoutes => `${navRoutes.rootRoute}/${profilePubKey}`,
+    });
+  };
+
+  const handleEntryClick = (itemId: string, itemType: EntityTypes) => {
+    if (itemType === EntityTypes.POST) {
+      navigateTo?.({
+        appName: '@akashaorg/app-akasha-integration',
+        getNavigationUrl: navRoutes => `${navRoutes.Post}/${itemId}`,
+      });
+    } else if (itemType === EntityTypes.REPLY) {
+      navigateTo?.({
+        appName: '@akashaorg/app-akasha-integration',
+        getNavigationUrl: navRoutes => `${navRoutes.Reply}/${itemId}`,
+      });
+    }
+  };
+
+  const labels = ['New', 'All'];
+
+  const handleTopMenuClick = () => {
+    setShowMenu(!showMenu);
+  };
+
+  const markAllAsRead = () => {
+    // do something
+    unreadNotifications.map(notif => {
+      //  markAsRead.mutate(notif.id);
+      if (notif.read === undefined) {
+        markAsRead.mutate(notif.id);
+      }
+    });
+    setShowMenu(!showMenu);
+
+    setMessage('Marked all as read successfully.');
+    setMessageType('success');
+    setShowFeedback(true);
+  };
+
+  const redirectToSettingsPage = () => {
+    // go to customization page
+    return navigateTo?.({
+      appName: '@akashaorg/app-notifications',
+      getNavigationUrl: () => routes[SETTINGS_PAGE],
+    });
+  };
+
+  const dropDownActions: ListProps['items'] = [
+    {
+      label: 'Mark all as read',
+      icon: 'CheckCircleIcon',
+      onClick: () => markAllAsRead(),
+    },
+    {
+      label: 'Settings',
+      icon: 'Cog8ToothIcon',
+      onClick: () => redirectToSettingsPage(),
+    },
+  ];
+
+  return (
+    <>
+      <Card
+        direction="row"
+        elevation={'1'}
+        radius={16}
+        padding={{ x: 0, y: 8 }}
+        customStyle="pb-16"
+      >
+        <div className={tw('py-4 relative w-full')}>
+          <Text variant="h5" align="center">
+            Notifications
```

Comment:
add also the translation utility useTranslation to labels, you can check out other apps to see how, import import { useTranslation } from 'react-i18next';
const { t } = useTranslation('app-notifications');
then in the html <text>{t('Notifications')}</text>

Reply:
I have added it.